### DISCUSSION TITLE
Multilanguage switch

### DISCUSSION THREAD

**Discussion by Feezex** [jglim/CaesarSuite#51] [Ideas]
Id wish tool runs not only in english ^^ Yes i already did it but its not a good realization - by searching all text and butons in code.
If there any chance to store language in separate dll so we can quickly add and update languages content ?
:+1: :+1: :+1:

**Comment by jglim** [comment]
It's a good suggestion, and .NET also has proper support for localization. 

I might look into this at a later date as I do not yet know of a good way to crowdsource language data
